lighthouse-viewer/test/viewer-test-pptr.js

```diff
@@ -79,7 +79,7 @@ describe('Lighthouse Viewer', () => {
     if (pageErrors.length > 0) console.error(pageErrors);
 
     await Promise.all([
-      new Promise(resolve => server.close(resolve)),
+      server.close(),
```

Comment:
Another brainteaser!

`server.close` doesn't take any parameters. This file is not typechecked, so this wasn't caught. If this were a raw `http.Server` object, it would work as written. This mistake happened because of the StaticServer refactor.

The brainteaser is... why did this ever work? I suppose before, Jest simply gave up on some point and shutdown. But now it will fail if `afterPass` hangs.